src/index.html

```diff
@@ -22,6 +22,394 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a
+          href="#"
+          class="header__logo-link"
+        >
+          <img
+            src="images/logo.png"
+            alt="header__logo"
+            class="logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="nav__list-item">
+              <a
+                href="#"
+                class="nav__item"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="main">
+      <div
+        class="container"
+        data-qa="card"
+      >
+        <img
+          class="container__image"
+          src="images/imac.jpeg"
```

Comment:
The `alt` attribute should be more descriptive. Instead of 'Apple Device(imac)', consider something like 'Apple iMac 27" with Retina 5K display' to provide better context.